st2common/st2common/content/bootstrap.py

```diff
@@ -219,8 +257,23 @@ def register_content():
     if cfg.CONF.register.policies:
         register_policies()
 
+    if cfg.CONF.register.setup_virtualenvs:
+        setup_virtualenvs()
+
 
 def setup(argv):
+    if '--register-setup-virtualenvs' in argv:
+        # If setup-virtualenvs option is used we also want to setup config for action runner since
```

Comment:
This is a hack, but it's sadly not possible to do it differently because args need to be parsed before we can check `cfg.CONF...`.

And we can't just always use action runner config since `st2actions` doesn't necessary be installed on every host (well, that will always be true with the new packages, but if someone does something funky this would not always be true).